astro/src/content/docs/get-started/core-concepts/users.mdx

```diff
@@ -99,14 +99,28 @@ These are the default fields that are available when creating a user. You can ad
       Then only the setup password message will be sent to the user. Setting up the password using the message sent during this user create operation will verify the user’s email address or phone number if it is not already verified.
     </Aside>
     <Aside>
-      This is only available if the [Setup password option](/docs/get-started/core-concepts/tenants#template-settings) in the tenant Template settings is set up.
+      <InlineField>Send password setup to</InlineField> is only available if the following tenant settings are:
```

Comment:
NOTE that the absence of template settings does NOT prevent this from showing up on the form, it just prevents user creation from working (the User API will reject the request when it validates).

Not sure if that distinction should be made here.